model/project_parser_s3.go

```diff
@@ -12,6 +12,9 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 )
 
+// sizeLimit is a hard limit on parser project size.
+const sizeLimit = 1024 * 1024 * 50
```

Comment:
Do we need a hard-coded size limit? I thought the parser project size was supposed to be configurable from the admin settings so we could incrementally raise it in a controlled way.

I believe the actual current limitation is below 50 MB, so we would likely want to adjust this value over time, alongside tuning the other parser project settings.

Reply:
Updated this to be configurable in admin settings. 

I'm not aware of any current limitation beyond the 16MB document limit though:

> I believe the actual current limitation is below 50 MB (depending on the concurrency setting)

Where can I find this?